openapi/code/package.go

```diff
@@ -162,40 +134,80 @@ func (pkg *Package) schemaToEntity(s *openapi.Schema, path []string, hasName boo
 	if s.JsonPath != "" {
 		processedEntities[s.JsonPath] = e
 	}
-	// pull embedded types up, if they can be defined at package level
-	if s.IsDefinable() && !hasName {
```

Comment:
Why not using `IsDefinable`? I also don't see other usages if that function. We may want to remove it.

Reply:
I think keeping the function makes the code less readable. By having the code like below, you know in one go that (i) it only applies to `object` and `enum`, and more importantly (ii) that the inner if statement is handling all the cases and is correct. 

```go
if (s.IsObject() || s.IsEnum()) && !hasName {
		fieldType := "enum"
		if s.IsObject() {
			fieldType = "object"
		}
```

I didn't realize that the function was only called there so yes, I think it's better to remove it.